.prettierignore

```diff
@@ -1 +1,3 @@
 pnpm-lock.yaml
+worker-configuration.d.ts
+runtime.d.ts
```

Comment:
this one is no longer needed 🙂

```suggestion
```

Reply:
29ae486edaaab32bac00a02ac26cbb835440ed4b